Exercise_A/index.html

```diff
@@ -10,8 +10,72 @@
       name="viewport"
       content="width=device-width, initial-scale=1"
     />
+    <link
+    rel="stylesheet"
+    href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+    crossorigin="anonymous"
+  />
+  <link rel="stylesheet" href="style.css">
   </head>

```

Comment:
Great implementation of the bootstrap components Candice well done!
Great use of the inbuilt bootstrap utilities